docs/components/table.md

```diff
@@ -195,6 +195,125 @@ export default {
+| `colIndex`| The zero-based index of the cell within a row
+
+<template>
+  <KTable
+    :options="tableOptionsCellAttrs"
+    :cellAttrs="cellAttrsFn"
+    />
+</template>
+
+```vue
+<template>
+  <KTable
+    :options="tableOptions"
+    :cellAttrs="cellAttrsFn"
+    />
+</template>
+<script>
+export default {
+  data() {
+    return {
+      row: null,
+      eventType: '',
+      tableOptions: {
+        headers: [
+          { label: 'Name', key: 'name' },
+          { label: 'Company', key: 'company' },
+          { label: 'Description', key: 'description' }
+        ],
+        data: [
+          {
+            name: 'SageMaker',
+            company: 'Amazon',
+            description: 'Amazon SageMaker is a fully-managed service that enables developers and data scientists to quickly and easily build, train, and deploy machine learning models at any scale. Amazon SageMaker removes all the barriers that typically slow down developers who want to use machine learning.',
+          },
+          {
+            name: 'Azure Machine Learning Studio',
+            company: 'Microsoft',
+            description: 'Azure Machine Learning Studio is a GUI-based integrated development environment for constructing and operationalizing Machine Learning workflow on Azure.',
+          },
+          {
+            name: 'IBM Watson Machine Learning',
+            company: 'IBM',
+            description: 'IBM Watson Studio accelerates the machine and deep learning workflows required to infuse AI into your business to drive innovation.',
+          },
+          {
+            name: 'TensorFlow',
+            company: 'Google',
+            description: 'TensorFlow is an open source software library for numerical computation using data flow graphs.',
+          },
+        ]
+      }
+    }
+  },
+  methods: {
+    cellAttrsFn ({ headerKey, row, rowIndex, colIndex }) {
+      /**
+       * Sets cell background color based on data returned in
+       * the row parameter and the index of the cell
+       */
+      const backgroundColor = () => {
+        if (row.company && row.company === 'Google') {
+          if (colIndex === 0) {
+            return '#4285F4'
+          } else if (colIndex === 1) {
+            return '#DB4437'
+          } else {
+            return '#F4B400'
+          }
+        }
+
+        return ''
+      }
+
+      /**
+       * Sets cell max-width based on the column headerKey
+       */
+      const colWidth = () => {
+        if (headerKey === 'description') {
+          return '50ch'
+        } else if (headerKey === 'name') {
+          return '22ch'
+        }
+
+        return '25ch'
+      }
+
+      /**
+       * Returns an object of attributes to be applied to cells
+       */
+      return {
+        class: {
+          'truncate': headerKey === 'description' || headerKey === 'name',
+        },
+        'datatest-id': `row-${rowIndex + 1}-col-${headerKey}`,
+        style : {
```

Comment:
```suggestion
        style: {
```

Super tiny nit: remove unnecessary space.

Reply:
Fixed in latest commit along with other requested changes, thanks for catching it!